src/Hyperion/PrintReport.hs

```diff
@@ -29,6 +28,6 @@ formatReport report =
       | x > 1e3 = show2decs (x/1e3) ++ "us"
       | otherwise = show2decs x ++ "ns"
 
-printReports :: HashMap Text Report -> IO ()
-printReports report = do
-  putDoc $ foldMap formatReport (elems report)
+printReports :: [Report] -> IO ()
```

Comment:
The idea is not to have an ordered list, but rather show that we do not care about the keys. I could go with a `Set Report` but this seems overkill.